mmpose/evaluation/functional/nms.py

```diff
@@ -249,3 +249,75 @@ def soft_oks_nms(kpts_db: List[dict],
     keep = keep[:keep_cnt]
 
     return keep
+
+
+def nearby_joints_nms(
+    kpts_db,
+    dist_thr,
+    num_nearby_joints_thr=None,
+    score_per_joint=False,
+    max_dets: int = 30,
+):
+    """Nearby joints NMS implementations.
```

Comment:
Could you please add a description or paper reference of the algorithm?

Reply:
And it would be better to add type hints as well.

Reply:
This function is modified from the implementation in the official code. The reference to their repo has been added